packages/manager/modules/pci/src/projects/project/users/add/user-add/user-add.html

```diff
@@ -6,16 +6,17 @@
     data-secondary-label="{{:: 'pci_projects_project_users_add_cancel_label' | translate }}"
     data-on-dismiss="$ctrl.goBack()"
     data-loading="$ctrl.isLoading">
-
-    <oui-field
-        data-label="{{:: 'pci_projects_project_users_add_description_label' | translate }}"
-        data-size="xl">
-        <input class="oui-input"
-            type="text"
-            id="description"
-            name="description"
-            ng-model="$ctrl.user.description"
-            maxlength="255"
-            required>
-    </oui-field>
+    <form novalidate name="user-add-form">
```

Comment:
`form` should encapsulate the `oui-modal` (containing the submit button https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-modal/src/modal.html#L36) 
Example : https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/streams/stream/backlogRetention/backlog-retention.html#L1